src/vm/docs/mod.rs

```diff
@@ -233,6 +233,17 @@ const SQRTI_API: SimpleFunctionAPI = SimpleFunctionAPI {
 "
 };
 
+const LOG2_API: SimpleFunctionAPI = SimpleFunctionAPI {
+    name: None,
+    signature: "(log2 n)",
+    description: "Returns the power to which the number 2 must be raised to to obtain the value `n`, rounded down to the nearest integer.  Fails on a negative numbers.",
```

Comment:
Also known as the _characteristic_ of the logarithm.

Reply:
Should be _fails on a non-positive integer_ as the logarithm of zero is undefined.